### ISSUE TITLE
Screen is stucked, after keyboard is disappeare

### ISSUE THREAD

**Issue by umesh-nikam** [driftyco/cordova-plugin-keyboard#155]
## Your Environment
* Plugin version:  2.1.3
* Platform: iOS
* OS version: 13.7
* Device manufacturer / model: iPhone 11 (iPhone12,1)
* Cordova version (`cordova -v`): 8.1.2
* Cordova platform version (`cordova platform ls`): 5.0.0

## Expected Behavior
After enter value in Input dialog, Screen should be resized after keyboard is disappeared

## Actual Behavior
Screen is stucked after keyboard is disappeared.

![image (2)](https://user-images.githubusercontent.com/46701728/96596340-3a153080-130a-11eb-8f0c-fbac72ac0300.png)

## Steps to Reproduce

1. Open Input dialog
2. Input value & close dialog
3. White area behind keyboard remain as it is.

## Context
We are trying to create gallery by clicking "New" button in top right corner,
When we click "New" button, it open input dialog, we input gallery name & click create button on dialog & It will close input dialog.
But area behind keyboard is still remain as it is.

**Comment by wswoodruff** [comment]
Yup, I'm also experiencing this issue. I left details for my setup in this comment on a related thread https://github.com/apache/cordova-ios/issues/417#issuecomment-767149076
